### PR TITLE
Update katello 4.1 to use pulpcore 3.14

### PR DESCRIPTION
**Pull request by jjeffers**

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
Could you fix the merge conflict and fix the typo in the commit message?

File: vagrant/config/versions.yaml [hunk 1]
```diff
@@ -68,7 +68,8 @@ installers:
 
   - foreman: '2.5'
     katello: '4.1'
-    pulpcore: '3.11'
+    pulp: '2.21'
```
Comment (ekohl) [inline vagrant/config/versions.yaml:71]: This looks like a bad rebase that ends up reverting ab476516d1cf9809d9bc02c6456bdbebbf40c2df
```suggestion
```

**Review by ekohl — CHANGES_REQUESTED** [review]

**Review by ekohl — APPROVED** [review]
Code wise :+1: but this is just one piece of the puzzle. Last time we merged forklift too soon and broke many things. Let's wait until the packaging and core PRs are also ready. I also think we need to prepare an installer cherry pick PR.

**Comment by jjeffers** [comment]
> Code wise  but this is just one piece of the puzzle. Last time we merged forklift too soon and broke many things. Let's wait until the packaging and core PRs are also ready. I also think we need to prepare an installer cherry pick PR.

Ok understood.

**Comment by evgeni** [comment]
Everything else is ready, so let's do it!

**Pull request merged**